public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -0,0 +1,466 @@
+        super(props);
+        this.state = {
+            isCopeSurvey: false,
+            survey: null,
+            surveyPdfUrl: '',
+            searchWord: this.props.searchWord,
+            extraQuestionConceptIds: [],
+            showAnswer: {},
+            questions: [],
+            loading: true,
+            surveyVersions: [],
+            showStatement: false,
+        };
+        this.changeResults = this.changeResults.bind(this);
+        this.closePopUp = this.closePopUp.bind(this);
+        this.showCopeStatement = this.showCopeStatement.bind(this);
+    }
+
+    componentDidMount() {
+        this.fetchSurvey(this.props.domainId);
+    }
+
+    changeResults(val) {
+        this.fetchSurvey(val);
+    }
+
+    closePopUp() {
+        this.setState({showStatement: false});
+    }
+
+    showCopeStatement() {
+        this.setState({showStatement: true});
+    }
+
+    fetchSurvey(domain) {
+        const {domainId} = this.props;
+        const {searchWord} = this.state;
+        let fetchDomain = domainId;
+        if (domain && domainId !== domain) {
+            fetchDomain = domain;
+        }
+        api.getDomainTotals(searchWord, 1, 1).then(
+                (data: any) => {
+                  data.surveyModules.forEach(survey => {
+                    const surveyRoute = survey.name.split(' ').join('-').toLowerCase();
+                    if (surveyRoute.indexOf('(cope)') > -1) {
+                        if (fetchDomain && surveyRoute.indexOf(fetchDomain) > -1) {
+                            localStorage.setItem('surveyModule', JSON.stringify(survey));
+                            this.setSurvey(JSON.stringify(survey));
+                        }
+                    } else {
+                        if (surveyRoute === fetchDomain) {
+                            localStorage.setItem('surveyModule', JSON.stringify(survey));
+                            this.setSurvey(JSON.stringify(survey));
+                        }
+                    }
+                  });
+                });
+    }
+
+    setSurvey(surveyObj) {
+        if (surveyObj) {
+            const survey = JSON.parse(surveyObj);
+            const surveyConceptId = survey.conceptId;
+            let surveyPdfUrl = '';
+            if (surveyConceptId === 43528895) {
+                surveyPdfUrl = '/assets/surveys/' +
+                'Health Care Access Utilization'.split(' ').join('_') + '.pdf';
+            } else {
+                surveyPdfUrl = '/assets/surveys/' + survey.name.split(' ').join('_') + '.pdf';
+            }
+            let extraConcepts = [];
+            if (surveyConceptId === 43528698) {
+                extraConcepts = ['43528515', '1384639', '43528634', '43528761', '43529158', '43529767', '43529272', '43529217', '702786', '43529966', '43529638', '43528764', '43528763', '43528649', '43528651', '43528650', '43528765'];
+            }
+            const copeFlag = surveyConceptId === 1333342 ? true : false;
```

Comment:
Nit: condition can be simplified:
```suggestion
            const copeFlag = surveyConceptId === 1333342;
```

Reply:
Changed this thanks.